js/server/tests/aql/aql-functions-string.js

```diff
@@ -626,6 +626,27 @@ function ahuacatlStringFunctionsTestSuite () {
       assertQueryError(errors.ERROR_QUERY_FUNCTION_ARGUMENT_NUMBER_MISMATCH.code, 'RETURN REGEX_SPLIT("test", "meow", "foo", "bar", "git")');
     },
 
+
+  // //////////////////////////////////////////////////////////////////////////////
+// / @brief test RegexMatches
+// //////////////////////////////////////////////////////////////////////////////
+    testToRegexMatchesValues: function () {
+      [ 
+        [ "hypertext language, programming", "[\s, ]+", true, [" "] ],
```

Comment:
Here again, please add some more meaningful tests.
Ideally we should have lots and lots of tests for `REGEX_MATCHES`, because it has so many use cases.
I don't see any tests for case-insensitive matching or for case-sensitive matching.
We should also have tests with multi-byte character strings. I also see no tests for capturing matches in parentheses.
These should all be added, because these are important features.